compiler/parser/parser.peg

```diff
@@ -546,13 +547,30 @@ MergeProc
     }
 
 OverProc
-  = "over"i _ exprs:Exprs {
-	  RETURN(MAP("kind":"Over", "exprs":exprs))
+  = ScopedOver
+  / "over"i _ exprs:Exprs {
+	  RETURN(MAP("kind":"Over", "exprs":exprs, "scope":NULL))
+    }
+
+ScopedOver
+  = "over"i _ exprs:Exprs __ scope:Scope {
```

Comment:
Nit: Crop the Scope nonterminal since it doesn't appear to be used anywhere else.
```suggestion
  = "over"i _ exprs:Exprs __ "=>" __ "(" __ scope:Sequential __ ")" {
```